pyroute2/netlink/core.py

```diff
@@ -165,12 +165,14 @@ def __init__(
         flags=os.O_CREAT,
         libc=None,
         groups=0,
+        use_event_loop=False,
```

Comment:
perhaps that this option should be accepted in IPRoute? We have a lot of code using the "sync" class

Reply:
I was two minutes too late

Reply:
Good catch, thanks! And it's never too late :)

I will add that definitely.